.eslintrc

```diff
@@ -16,7 +16,8 @@
   "rules": {
     "no-empty": ["error", { "allowEmptyCatch": true }],
     "prefer-destructuring": ["warn", { "object": true, "array": false }],
-    "no-dupe-class-members": ["warn"],
+    "no-dupe-class-members": "off",
+    "no-redeclare": "off",
```

Comment:
not pretty sure why we should add this rule that can lead to confusion to where the variable is declared and initialised